test/registry/registry_tests.py

```diff
@@ -872,10 +873,18 @@ def test_image_replication_empty_layers(
             credentials=credentials,
         )
 
+<<<<<<< HEAD
```

Comment:
Looks like a merge diff got left in here.

Reply:
Yeah, these were removed in a followup change. Since all of this will be squashed on merge, I figured it was easier to leave it in then rebase

Reply:
Sounds good! I'll close these ones